contributing.md

```diff
@@ -43,8 +43,16 @@ here](https://join.slack.com/t/bazelbuild/shared_invite/enqtndyyodaxndy4mjqxlwu1
    takes too long (e.g. reviewer is unresponsive), please also send an email to
    [bazel-dev@googlegroups.com](mailto:bazel-dev@googlegroups.com).
 1. A Bazel maintainer at Google will apply the patch to our internal version
-   control system. The patch is exported as a Git commit, at which point the
-   GitHub pull request is closed.
+   control system.
+
+   This triggers [internal presubmit checks](https://bazel.build/governance.html#are-you-done-open-sourcing-bazel)
+   that may suggest more changes. If you haven't expressed a preference, the
+   submitter will add "trivial" changes (like
+   [linting](https://en.wikipedia.org/wiki/Lint_(software))) that don't affect
+   design. If you're concerned about this difference, let the reviewer know and
+   they'll communicate on the PR what further changes are requested. After
```

Comment:
1. How the author communicates this preference to reviewer/merger? I think we should specify that.
2.  I am concerned of the burden on the reviewer to communicate changes required due to internal process. Unfortunately internal tools do not provide that info in easily externalizable format...  My suggestion would be for the reviewer to do the changes required and then show the changed code to the original author for final approval before committing (in the case the author prefers that) - wdyt?